martin/src/pg/query_tables.rs

```diff
@@ -204,7 +215,7 @@ FROM (
   FROM
     {schema}.{table}
   WHERE
-    {geometry_column} && ST_Transform({bbox_search}, {srid})
```

Comment:
@nyurik when we looked over this yesterday, we overlooked this part. We are already reprojecting..
Therefore what we discussed is moot